platform/security/interceptor/security-interceptor-guest/src/main/java/org/codice/ddf/security/interceptor/GuestInterceptor.java

```diff
@@ -387,16 +398,19 @@ public void handleMessage(SoapMessage message) throws Fault {
                 if (policyRequirementsSupported || overrideEndpointPolicies) {
                     LOGGER.debug("Creating guest security token.");
                     if (soapFactory != null) {
-                        HttpServletRequest request = (HttpServletRequest) message
-                                .get(AbstractHTTPDestination.HTTP_REQUEST);
-                        createSecurityToken(version, soapFactory, securityHeader,
+                        HttpServletRequest request = (HttpServletRequest) message.get(
+                                AbstractHTTPDestination.HTTP_REQUEST);
+                        createSecurityToken(version,
+                                soapFactory,
+                                securityHeader,
                                 request.getRemoteAddr());
                         try {
                             // Add security header to SOAP message
-                            soapMessage.getSOAPHeader().addChildElement(securityHeader);
+                            soapMessage.getSOAPHeader()
+                                    .addChildElement(securityHeader);
                         } catch (SOAPException e) {
-                            LOGGER.error("Issue when adding security header to SOAP message:" + e
-                                    .getMessage());
+                            LOGGER.error("Issue when adding security header to SOAP message:"
```

Comment:
Should use interpolation here instead of concatenation. (I know it was wrong before.)